test/read_mixin_test.rb

```diff
@@ -40,6 +40,24 @@
       assert_equal @post_records, Post.find_by_sql('SELECT * FROM `posts` WHERE (id = 1)')
     end
 
+    it "sends an ActiveSupport::Notification" do
+      notification_received = false
+      record_count = 0
+      class_name = ""
+      ActiveSupport::Notifications.subscribe "instantiation.active_record" do |*args|
+        event = ActiveSupport::Notifications::Event.new(*args)
+        record_count = event.payload[:record_count]
+        class_name = event.payload[:class_name]
+        notification_received = true
+      end
```

Comment:
might want to unsubscribe after done. see Temporary Subscriptions on https://api.rubyonrails.org/v4.2.5/classes/ActiveSupport/Notifications.html